### PR TITLE
feat(pix-machine): add how to install and use pix-machine

### PR DESCRIPTION
**Pull request by edumaciel10**

### CODE REVIEW

**Review by daniloab — COMMENTED** [review]
prefer prints in ptbr

File: docs/pix-machine/how-to-install-pix-machine.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Com o ID da impressora copiado, abra o aplicativo da maquininha e cole o ID no campo `ID da impressora` e clique em `Salvar`.
+
+![Maquininha Setup](__assets__/maquininha-setup.png)
+
+Pronto! Sua maquininha está pronta para ser usada.
+
+## Como testar se minha maquininha está funcionando
+
+Para testar se sua maquininha está funcionando, crie uma cobrança qualquer na plataforma OpenPix e veja se a impressão foi feita corretamente.
```
Comment (criskell) [inline docs/pix-machine/how-to-install-pix-machine.md:51]: linking the step-by-step instructions for creating a charge would be cool

**Review by criskell — APPROVED** [review]

**Review by jgcmarins — APPROVED** [review]
LGTM

**Review by sibelius — COMMENTED** [review]
prefer portuguese to images

**Pull request merged**